base/containers/checked_iterators.h

```diff
@@ -267,6 +269,27 @@ struct pointer_traits<::base::CheckedContiguousIterator<T>> {
 };
 
 }  // namespace std
+#elif defined(STARBOARD)
```

Comment:
I'm not sure this is necessary. See the upstream fix: https://gcc.gnu.org/pipermail/libstdc++-cvs/2020q2/031260.html. But it's ok to use it as a workaround for now. 
Can you create a TODO for removing this when we upgrade to c++20?